core/gesture.ts

```diff
@@ -169,10 +169,10 @@ export class Gesture {
 
   /**
    * @param e The event that kicked off this gesture.
-   * @param creatorWorkspace The workspace that created this gesture and has a
+   * @param creatorWorkspace_ The workspace that created this gesture and has a
    *     reference to it.
    */
-  constructor(e: Event, private readonly creatorWorkspace: WorkspaceSvg) {
+  constructor(e: Event, private readonly creatorWorkspace_: WorkspaceSvg) {
```

Comment:
Was the old name conflicting or something? I think we're generally not wanting to use underscores to indicate private-ness.

Reply:
We had a test checking for `creatorWorkspace_` idk why I decided to fix it this way. Reverted this and modified the test instead.